tools/realsense-viewer/openvino-face-detection.cpp

```diff
@@ -225,6 +226,7 @@ class openvino_face_detection : public post_processing_worker_filter
         }
         // A depth frame is optional: if not enabled, we won't get it, and we simply won't provide depth info...
         auto df = fs.get_depth_frame();
+        if (!fs && f.get_profile().stream_name() != "Depth") df = f;
```

Comment:
Most probably `f.get_profile().stream_name() == "Depth`

Reply:
actually this condition is not relevant at all : when single frame is received it must be RGB.
auto df = fs.get_depth_frame() returns NULL when single frame is received and that's all what we need for depth (no exception is thrown)